docs/people-and-groups/managing.md

```diff
@@ -1,12 +1,16 @@
 ---
-title: Managing people and groups
+title: People and groups
 redirect_from:
   - /docs/latest/administration-guide/04-managing-users
 ---
 
-# Managing people and groups
+# People and groups
 
-To start managing people:
+People can have [accounts](#creating-an-account) in Metabase, and those accounts can be members of [groups](#groups). These groups are used to define [permissions](../permissions/introduction.md). People can be in multiple groups.
```

Comment:
I wonder if we should add some sort of note that's like, "btw there are also Metabase store accounts that are entirely separate from Metabase instance accounts. go check out those other docs"